src/components/Tabs/Bar.js

```diff
@@ -58,6 +58,7 @@ const Bar = ({ children, className }) => {
       css={css`
         display: flex;
         width: 100%;
+        border-bottom: 1px solid var(--color-neutrals-300);
```

Comment:
Can we also supply a color for the dark theme? I think we were using `var(--color-dark-300)` before.

Reply:
good catch, i'm trying out --divider-color which is also used by the right rail, do you think it's too subtle? also realized I'd lost the button underline when selected so I added that back